doc/templates/member-associate-example.md

```diff
@@ -1,26 +1,27 @@
-title: GivenName - Surname (Required)
-date: YYYYMMDD !write the date that you joined (Required)
-joined_date: YYYYMMDD !write the date that you joined (Required)
-category: members (if member) or alumni (if alumni) BE CAREFUL: write it exactly as you see it
-id: (Required)
-givenname: (Required)
-surname: (Required)
-email: (Required)
-office_phone: (Optional)
-mobile_phone:  (Optional)
-office_address: (Optional)
-postal_address: (Optional)
-web_site: (Optional - URL)
-web_log: (Optional - URL)
-github: (Optional - just the handle, e.g. dspinellis)
-twitter: (Optional - just the handle, e.g. CoolSWEng)
-linkedin: (Optional - just the handle, e.g. dspinellis)
-photo: filename.jpg (Optional - A 50-100kb JPEG is recommended)
-phd_title: (Required for PhD students)
-phd_supervisor: member-id (Optional)
-phd_startdate: YYYYMMDD (Required for PhD students)
-phd_enddate: YYYYMMDD (Required for PhD students)
-phd_completed: 0(false) or 1(true) (Required)
-phd_summary: (Required for PhD students) !BE CAREFUL: write the summary in one line without break lines
-
-WRITE HERE YOUR SHORT CV EITHER IN HTML FORM OR IN MARKDOWN FORM (Required)
+title: GivenName - Surname (Required)
+date: YYYYMMDD !write the date that you joined (Required)
+joined_date: YYYYMMDD !write the date that you joined (Required)
+category: members (if member) or alumni (if alumni) BE CAREFUL: write it exactly as you see it
+member_category: Researchers or Associate Researchers or Senior Researchers or Faculty Members BE CAREFUL: write it exactly as you see it
+id: (Required)
+givenname: (Required)
+surname: (Required)
+email: (Required)
+office_phone: (Optional)
+mobile_phone:  (Optional)
+office_address: (Optional)
+postal_address: (Optional)
+web_site: (Optional - URL)
+web_log: (Optional - URL)
+github: (Optional - just the handle, e.g. dspinellis)
+twitter: (Optional - just the handle, e.g. CoolSWEng)
+linkedin: (Optional - just the handle, e.g. dspinellis)
+photo: filename.jpg (Optional - A 50-100kb JPEG is recommended)
+phd_title: (Required for PhD students)
+phd_supervisor: member-id (Optional)
+phd_startdate: YYYYMMDD (Required for PhD students)
+phd_enddate: YYYYMMDD (Required for PhD students)
+phd_completed: 0(false) or 1(true) (Required)
+phd_summary: (Required for PhD students) !BE CAREFUL: write the summary in one line without break lines
+
+WRITE HERE YOUR SHORT CV EITHER IN HTML FORM OR IN MARKDOWN FORM (Required)

```

Comment:
These changes shouldn't exist, right? Revert and (and configure your editor to avoid creating such changes).